<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormScore</class>
 <widget class="QWidget" name="FormScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>points</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/image/snake.jpeg</normalon>
   </iconset>
  </property>
  <widget class="QTableWidget" name="ShowScore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>811</width>
     <height>481</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lastname</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> Family</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_username">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> Score</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="score">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_score">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tartib">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Up</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Down</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="All">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
